shm/Ship.cpp

```diff
@@ -15,11 +17,11 @@ Ship::Ship(int maxCrew, int speed, size_t id, std::shared_ptr<Time>& time)
     : Ship(0, maxCrew, speed, "", id, time) {
 }
 
-Cargo* Ship::getCargo(size_t index) const {
-    if (index > cargo_.size() || index == 0) {
+CargoPtr Ship::getCargo(size_t index) const {
```

Comment:
It looks like `getCargo()` method from Ship class and Store are pretty similar and they could be also moved to StockManagement class. What do you think? Of course it requires to add one more parameter to list of arguments.